tests/test_edges/test_edge_population.py

```diff
@@ -0,0 +1,646 @@
+import sys
```

Comment:
while I'm not against splitting the tests of edges.py and nodes.py, this needs to be done in it's own pull request. 

We would also prefer to keep the git blame of the split files intact, and while we're splitting files, we could consider splitting the `nodes.py` and `edges.py`, too.